CONTRIBUTING.rst

```diff
@@ -32,11 +32,6 @@ Setting Up for Local Development
 
     # After activating your virtualenv
     $ pip install -r dev-requirements.txt
-    $ npm install -g check_api
-
-3. Install apispec in develop mode. ::
```

Comment:
This is no longer necessary because `pip install -r dev-requirements.txt` takes care of this.